Packs/FeedCrowdstrikeFalconIntel/Integrations/CrowdStrikeIndicatorFeed/CrowdStrikeIndicatorFeed.py

```diff
@@ -55,7 +55,7 @@ def __init__(self, client_id, client_secret, base_url, include_deleted, type, li
         self.limit = limit
 
     def http_request(self, method, url_suffix=None, full_url=None, headers=None, params=None, data=None,
-                     timeout=10, auth=None) -> dict:
+                     timeout=120, auth=None) -> dict:
```

Comment:
120 seconds or mili?

Reply:
120 seconds, I changed it since the request failed on timeout when I asked 10,000 indicators.

Reply:
let's change to 30. Let's also add to the additional info of the max fetch a disclaimer saying that big numbers might result in timeouts

Reply:
done